core/src/common/lwm2m_plaintext.c

```diff
@@ -184,7 +184,6 @@ static int PTDeserialiseResource(SerdesContext * serdesContext, Lwm2mTreeNode **
     {
         Lwm2m_Error("ERROR: Failed to set definition Object %d Resource %d\n", objectID, resourceID);
         return -1;
-    }
```

Comment:
Nope... but too lazy to redo the merge commit ;) What would happen if I squashed the commit into the merge commit?